core/assets/bundles/bundle.properties

```diff
@@ -673,4 +669,113 @@ tutorial.router = Factories need resources to function.\nCreate a router to spli
 tutorial.dagger = Link power nodes to the factory.\nOnce requirements are met, a mech will be created.\n\nCreate more drills, generators and conveyors as necessary.
 tutorial.battle = The[LIGHT_GRAY] enemy[] has revealed their core.\nDestroy it with your unit and dagger mechs.
 block.core.description = The most important building in the game.
+block.copper-wall.description = A cheap defensive block.\nUseful for protecting the core and turrets in the first few waves.
+block.copper-wall-large.description = A cheap defensive block.\nUseful for protecting the core and turrets in the first few waves.\nSpans multiple tiles.
+block.dense-alloy-wall.description = A standard defensive block.\nAdequate protection from enemies.
+block.dense-alloy-wall-large.description = A standard defensive block.\nAdequate protection from enemies.\nSpans multiple tiles.
+block.thorium-wall.description = A strong defensive block.\nGood protection from enemies.
+block.thorium-wall-large.description = A strong defensive block.\nGood protection from enemies.\nSpans multiple tiles.
+block.phase-wall.description = Not as strong as a thorium wall but will deflect bullets unless they are too powerful.
+block.phase-wall-large.description = Not as strong as a thorium wall but will deflect bullets unless they are too powerful.\nSpans multiple tiles.
+block.surge-wall.description = The strongest defensive block.\nHas a small chance of triggering lightning towards the attacker.
+block.surge-wall-large.description = The strongest defensive block.\nHas a small chance of triggering lightning towards the attacker.\nSpans multiple tiles.
+block.door.description = A small door that can be opened and closed by tapping on it.\nIf opened, enemies can shoot and move through.
+block.door-large.description = A large door that can be opened and closed by tapping on it.\nIf opened, enemies can shoot and move through.\nSpans multiple tiles.
+block.mend-projector.description = Periodically heals buildings in its vicinity.
+block.overdrive-projector.description = Increases the speed of nearby buildings like drills and conveyors.
+block.force-projector.description = Creates a hexagonal force field around itself, protecting buildings and units inside from damage through bullets.
+block.shock-mine.description = Damages enemies stepping on the mine. Not visible to the enemy.
```

Comment:
I'm not sure if "not visible by the enemy" is accurate. They _can_ see the mines in-game, but unit and ship auto-targeting won't work on them. "Nearly visible to the enemy." would be preferable.

Reply:
What about multiplayer?

Reply:
They can still be seen and destroyed, but there's no outline visible.